.github/workflows/e2e.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         CI_HETZNER_SACRIFICIAL_SERVER_ID: ${{ secrets.CI_HETZNER_SACRIFICIAL_SERVER_ID }}
         HETZNER_USER: ${{ secrets.HETZNER_USER }}
         HETZNER_PASSWORD: ${{ secrets.HETZNER_PASSWORD }}
-      run: foreman start
+      run: timeout 2400 foreman start
```

Comment:
```suggestion
      run: timeout 40m -- foreman start
```

?

Reply:
I changed as `timeout 40m foreman start`. `--` fails with the `timeout: failed to run command ‘--’: Permission denied` error.

> timeout doesn't strictly need this "--" because it treats any arguments after the duration as the command to run. The ﻿timeout command as implemented by GNU (common in most Linux distributions) does not require ﻿-- before the command to run. If ﻿-- is used, ﻿timeout mistakenly interprets it as part of the command to run, not as the end of options, hence the error message